src/service/communication.service.spec.ts

```diff
@@ -195,12 +196,8 @@ describe('BBHelpCommunicationService', () => {
       },
       origin: 'Other Source'
     };
-
-    spyOn(window.console, 'error').and.callFake(() => {
-      return;
-    });
     triggerEvent(event);
-    expect(window.console.error).toHaveBeenCalledWith('Event origin not supported.');
+    expect(commService.communicationAction.next).not.toHaveBeenCalled();
```

Comment:
This expectation could still hit the default case for the switch statement, which doesn't call `communicationAction.next`

Reply:
good catch